vendor/engines/projects/app/helpers/projects/application_helper.rb

```diff
@@ -0,0 +1,22 @@
+module Projects
+
+  module ApplicationHelper
+
+    def project_id_assignment_options(projects)
+      blank_options + options_from_collection_for_select(projects, :id, :name)
+    end
+
+    private
+
+    def blank_options
```

Comment:
It might be a good idea to give this a more specific name. Private methods in modules might get weird, especially with these helpers.

```ruby
module A
  private
  def priv
    "a"
  end
end

module B
  private
  def priv
    "b"
  end
end

class Testing
  include A
  include B
  def test
    priv
  end
end

> Testing.new.test
=> "b"
```